### DISCUSSION TITLE
New Python parser

### DISCUSSION THREAD

**Discussion by DavidMStraub** [FamilySearch/GEDCOM#218] [Show and tell]
Hello,

I've just released a new, MIT-licensed, Python parser for GEDCOM 7: see https://github.com/DavidMStraub/python-gedcom7. Comments & contributions welcome.

**Comment by clarkegj** [comment]
Can I post a link to this at gedcom.io/tools/ ?

**Reply by DavidMStraub** [reply]
Sure, that would be great, thanks.

**Comment by clarkegj** [comment]
check out  https://gedcom.io/tools/

**Reply by dthaler** [reply]
> check out https://gedcom.io/tools/

The site url for the python parser shows up as text on that page but isn't clickable, can you fix that @clarkegj ?

**Reply by DavidMStraub** [reply]
Fixed here: https://github.com/FamilySearch/GEDCOM.io/pull/57
